src/main/java/uk/gov/ida/verifyserviceprovider/configuration/EuropeanIdentityConfiguration.java

```diff
@@ -38,6 +44,11 @@ public EuropeanIdentityConfiguration(@JsonProperty("hubConnectorEntityId") Strin
         this.enabled = enabled;
         this.hubConnectorEntityId = hubConnectorEntityId;
         this.trustStoreConfiguration = trustStore;
+
+        Set<String> hubConnectorEntityIds = new LinkedHashSet<>();
+        Optional.ofNullable(acceptableHubConnectorEntityIds).ifPresent(hubConnectorEntityIds::addAll);
+        Optional.ofNullable(hubConnectorEntityId).ifPresent(hubConnectorEntityIds::add);
+        this.acceptableHubConnectorEntityIds = new LinkedList<>(hubConnectorEntityIds);
```

Comment:
Again, out of interest, why the LinkedList rather than a plain old vanilla List?

Reply:
I think the alternative is plain old vanilla ArrayList (as List is an interface). I think IntelliJ autocompletes/offers LinkedList preferentially (or maybe I imagined that). Again, no strong feelings, but it's been my go-to List implementation for a while.

Reply:
Cool! No complaints from me, just interested.